src/main/java/ch/njol/skript/expressions/ExprOnlinePlayersCount.java

```diff
@@ -0,0 +1,146 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.bukkitutil.PlayerUtils;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.ScriptLoader;
+import ch.njol.util.coll.CollectionUtils;
+import ch.njol.util.Kleenean;
+import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.bukkit.event.server.ServerListPingEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Online Players Count")
+@Description({"The count of online players. This can be changed in a <a href='events.html#server_list_ping'>server list ping</a> event only to show fake online players count.",
+		"'real online players count' returns the real count of online players always and can't be changed.",
+		"",
+		"Fake online players count requires PaperSpigot 1.12.2+."})
+@Examples({"on server list ping:",
+		"	# This will make the max players count 5 if there are 4 players online.",
+		"	set the fake max players count to (online players count + 1)"})
+@Since("INSERT VERSION")
+public class ExprOnlinePlayersCount extends SimpleExpression<Number> {
+
+	static {
```

Comment:
Use only one parser mark here

Reply:
check the code

Reply:
I hace checked it already and it doesn't need the second parser mark

Reply:
Check again :(
parse mark 0 = it will be 'real' players count outside of a server list ping event, otherwise 'fake' 
parse mark 1 = it is 'real' players count and can be used anywhere
parse mark 2 = it is 'fake' players count and can only be used in a server list ping event

Reply:
You'd need only one for the 'real' player count, and then just check whether it is or not real for the rest.

Reply:
no